### ISSUE TITLE
Neural Net (MLP)

### ISSUE THREAD

**Issue by natemcmaster** [natemcmaster/march-madness#3]
Just succeeded in getting accuracy up to 83.6%. 

Data - joined-pca.arff
MLP setting - 0.05 learning rate, 0.9 momentum, 23 hidden nodes (i), 60/20/20 split

**Comment by natemcmaster** [comment]
I've been playing around a lot more with attribute selection, PCA and others. It seems that MLP performs best on the data set. By tweaking and playing with parameters I can get up to 85% on a few runs, but on average the different settings get about 83% accuracy. 

Here is my best run:
https://gist.github.com/natemcmaster/3ceead3ebac3745e2a9f#file-best-mlp-txt-L6260
